test/test.py

```diff
@@ -1375,27 +1377,32 @@ def _test_console_script(self, package=False, wheel=False):
             command += " a\t"
             self.assertEqual(self.sh.run_command(command), "arg\r\n")
 
-    @unittest.skipIf(os.uname()[0] == "Darwin", "Skip test that fails on MacOS")
```

Comment:
These tests all pass in CI now, but I vaguely remember you were having trouble running them at some point. @kislyuk can you please pull this branch and confirm that the tests pass for you locally?

Reply:
Confirming that all tests pass on this branch for me.